src/ess/isissans/components.py

```diff
@@ -33,7 +28,7 @@ def apply_component_user_offsets_to_raw_data(
     data: RawData[ScatteringRunType],
     sample_offset: Optional[SampleOffset],
     detector_bank_offset: Optional[DetectorBankOffset],
-) -> RawDataWithComponentUserOffsets[ScatteringRunType]:
+) -> PatchedData[ScatteringRunType]:
```

Comment:
So we are adding missing coordinates, variances on the events, and are possibly adding offsets to some of the coordinates (in the case of ISIS data).

Either we find a name that covers all 3 well enough (I don't really get the above from `ConfiguredData`, but `PatchedData` is indeed quite general), or we should make them into separate steps?

I don't like making separate steps because it means that all of these steps could be optional, and we'd have to have e.g. a dummy step for applying position offsets in `loki`, just because it is needed in `isis`.

I bascially wanted a name that would mean "ready for the rest of the reduction workflow", but was never able to find a good name :-(

Reply:
How about just dropping the `Raw` prefix? This function would turn `RawData` into `Data`. But I still don't see what is so wrong with configured-data.

Reply:
I tried `ConfiguredAndCompleteData` but I still don't really like it. Now I'm thinking of `ConfiguredReducibleData` to hint that it has the necessary pieces to proceed with the rest of the reduction.

> But I still don't see what is so wrong with configured-data.

I think `Configured` captures the notion that some configurations were applied by the user, e.g. positions offsets, but not so much the fetching of the sample/source positions so that we can later perform conversion to wavelength. That's why I thought we were "patching" different parts of the NeXus file together. But I agree that `PatchedData` is just as vague.

Regarding why I thought we could be more specific in your PR was because it was to describe only the step where user offsets were applied to the data. Now that different things can happen in this step, we do need to go back to something more general.